src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx

```diff
@@ -226,67 +232,67 @@ const TokensMoveDialogForm = ({
-                ),
-                symbol: (selectedToken && selectedToken.symbol) || '???',
-              }}
-            />
-          </div>
-        )}
-      </DialogSection>
-      <DialogSection>
-        <Select options={domainOptions} label={MSG.to} name="toDomain" />
-        {values.toDomain !== undefined && !!values.tokenAddress && (
-          <div className={styles.domainPotBalance}>
-            <FormattedMessage
-              {...MSG.domainTokenAmount}
-              values={{
-                amount: (
-                  <Numeral
-                    appearance={{
-                      size: 'small',
-                      theme: 'grey',
-                    }}
-                    value={toDomainTokenBalance || 0}
-                    unit={getTokenDecimalsWithFallback(
-                      selectedToken && selectedToken.decimals,
-                    )}
-                    truncate={3}
-                  />
-                ),
-                symbol: (selectedToken && selectedToken.symbol) || '???',
-              }}
+        <div className={styles.domainSelects}>
+          <div>
+            <Select
+              options={domainOptions}
+              label={MSG.from}
+              name="fromDomain"
+              appearance={{ theme: 'grey' }}
             />
+            {!!tokenAddress && (
+              <div className={styles.domainPotBalance}>
+                <FormattedMessage
+                  {...MSG.domainTokenAmount}
+                  values={{
+                    amount: (
+                      <Numeral
+                        appearance={{
+                          size: 'small',
+                          theme: 'grey',
+                        }}
+                        value={fromDomainTokenBalance || 0}
+                        unit={getTokenDecimalsWithFallback(
+                          selectedToken && selectedToken.decimals,
+                        )}
+                        truncate={3}
+                      />
+                    ),
+                    symbol: (selectedToken && selectedToken.symbol) || '???',
+                  }}
+                />
+              </div>
+            )}
           </div>
-        )}
+          <Icon
+            className={styles.transferIcon}
+            name="circle-arrow-back"
+            title="Transfer"
+            appearance={{ size: 'medium' }}
+          />
+          <Select
+            options={domainOptions}
+            label={MSG.to}
+            name="toDomain"
+            appearance={{ theme: 'grey' }}
+          />
+        </div>
       </DialogSection>
       <DialogSection>
         <div className={styles.tokenAmount}>
           <div>
             <Input
               label={MSG.amount}
               name="amount"
-              appearance={{ theme: 'minimal', align: 'right' }}
+              appearance={{
+                theme: 'minimal',
+                align: 'right',
+                colorSchema: 'grey',
```

Comment:
Sorry if I made myself misunderstood, but this should have just been applied to the two select components and the one textarea.

The way you implemented it initially was correct:

![Screenshot from 2020-11-27 14-30-47](https://user-images.githubusercontent.com/1193222/100449795-3620bf00-30bd-11eb-81b4-91714c5d8481.png)

Just remove the `colorSchema` from here, and we're all good.